### PR TITLE
[Excel] (Custom functions) Web request article edits

### PR DESCRIPTION
**Pull request by mscharlock**
Updates XHR sample, fetch example, and I did some rearranging of information in this article

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ff5b7ce](https://github.com/OfficeDev/office-js-docs-pr/pull/1107/commits/ff5b7ce29b391a10389864760208255aefc59f56)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-xhrsecondfix/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906202003136718-1107%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Z2f3BoT6Sr9zmT%2F%2FrvjKiAPU2L33xypNfEeuKMv9uuc%3D&st=2019-06-20T19%3A59%3A18Z&se=2019-07-21T20%3A04%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-web-reqs.md [hunk 1]
```diff
@@ -1,146 +1,137 @@
...
+
+    xhttp.send();
+  });
+}
+```
 
 ### Fetch example
 
-In the following code sample, the `stockPriceStream` function uses a stock ticker symbol to get the price of a stock every 1000 milliseconds. For more details about this sample, see the [Custom functions tutorial](../tutorials/excel-tutorial-create-custom-functions.md#create-a-streaming-asynchronous-custom-function).
+It may be preferable to use the `Fetch` API over XHR for simplicity, as it avoids nested callbacks.
```
Comment (mscharlock) [inline docs/excel/custom-functions-web-reqs.md]:
```suggestion
Using `Fetch` avoids nested callbacks and may be preferable in some cases to XHR. 
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[04368e9](https://github.com/OfficeDev/office-js-docs-pr/pull/1107/commits/04368e91bdda6a4eaf8fbcfbfe2a4b99504650eb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-xhrsecondfix/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906202147406820-1107%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9uxWYJg4LmKp7hzJxzlfEmUYRX%2F%2BowufomsPOBDSQ%2Fc%3D&st=2019-06-20T21%3A43%3A43Z&se=2019-07-21T21%3A48%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c1d73aa](https://github.com/OfficeDev/office-js-docs-pr/pull/1107/commits/c1d73aa965ecf77ab0412f422c0d02d6687c34e2)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-3babf28f)

<a id="3babf28f"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\jfed-s, target branch: master, target branch commit: 130597550214134873340fa6f817bfe144713aac, source commit: c1d73aa965ecf77ab0412f422c0d02d6687c34e2.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906252012072998-1107%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=XO3kKFYeK39BE9JawmKY1nvEm7l7mXlrOPUgsP60BUk%3D&st=2019-06-25T20%3A07%3A20Z&se=2019-07-26T20%3A12%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c74501d](https://github.com/OfficeDev/office-js-docs-pr/pull/1107/commits/c74501daee5b361616c716c01cdd3d750d8fe992)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-42b29e70)

<a id="42b29e70"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\cyp1-s, target branch: master, target branch commit: c115b908e5823ba8bca15713d6c79acf7e2db44c, source commit: c74501daee5b361616c716c01cdd3d750d8fe992.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906260135112296-1107%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1MSw4LdVQmTnreNpBck7kzSswZzcuDesJUp%2FtHAFj%2FA%3D&st=2019-06-26T01%3A30%3A23Z&se=2019-07-27T01%3A35%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2542119](https://github.com/OfficeDev/office-js-docs-pr/pull/1107/commits/25421196ed1811d71dd3aad8a6583461d50f296d)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-724f4f5d)

<a id="724f4f5d"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\fdux-s, target branch: master, target branch commit: b4b91b7fb0a5f88cbc67ffe683274e08ba233213, source commit: 25421196ed1811d71dd3aad8a6583461d50f296d.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906272044418778-1107%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=I1RIKcLuD0qLbJCwXfoPIvYwFPi8BIp%2F15WU71gjSXc%3D&st=2019-06-27T20%3A39%3A53Z&se=2019-07-28T20%3A44%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[df88d88](https://github.com/OfficeDev/office-js-docs-pr/pull/1107/commits/df88d88c46260cfc2127954ce3cc55fc1c37659b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-xhrsecondfix/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |
[javascript/api/custom-functions-runtime/customfunctions.cancelableinvocation](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-xhrsecondfix/javascript/api/custom-functions-runtime/customfunctions.cancelableinvocation) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906272047592133-1107%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2BgTlQBFmPA1fe6ktxx49Cmr9BaBEsovBL%2BP91FNePk4%3D&st=2019-06-27T20%3A43%3A56Z&se=2019-07-28T20%3A48%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**